ehabi32/ehabi32.rst

```diff
@@ -1427,6 +1431,18 @@ The details are as follows:
 
     In general an unwinder must load *all* the machine registers listed in the VRS.
 
+   If Branch Target Identification mechanism is enabled, the unwinder
```

Comment:
A similar suggestion to the previous one:

If the Branch Target Identification mechanism is enabled, the unwinder may transfer control to the routine installed in VRS[r15] using a BTI setting instruction that requires a BTI clearing instruction at the destination.